src/pages/blog/a-puzzle-in-javascript-objects/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'A puzzle in JavaScript objects'
-description: ""
+description: "This is a puzzle you can run into if you’re not careful, and sometimes, despite your best efforts, even if you are careful. It happened to me when I had a settings object that I passed into a view. The view then did things. When I made a new view from the settings object, it looked like the view I had just cleared."
```

Comment:
I think we need a better description here. Something that teases the content, but doesn’t reveal it and makes you want to click.

This description sounds confusing and I have no idea what the article is going to be about

Reply:
Cool, I'm gonna have to test my understanding of the article, and let's see if I can come up with a description that summarizes it without giving away the sauce.